downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,7 +60,46 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
-#### CLI
+#### Test Framework
+
+- The test framework now supports for parallel execution of tests as an experimental feature. To activate the parallel test execution, simply use the `--parallel` flag with the `bal test` command.
+
+    ```ballerina
+    bal test --parallel
+    ```
+
+- APIs for mocking object resources is introduced. With these, an object resource can be stubbed to behave in a certain way, previously limited to test double. 
```

Comment:
Should we specifically mention resource functions in client objects?

Reply:
IMO, Mentioning "resource functions in the client objects" looks good. Shall we modify the example as follows? 
`main.bal`
```ballerina
public type Employee record {|
    readonly string id;
    string firstName;
    string lastName;
    string gender;
|};
EmpClient empClient = new ();
public client class EmpClient {
    map<Employee> employees;
    function init() {
        self.employees = {};
    }
    resource function get employee/[string id]() returns Employee? {
        return self.employees[id];
    }
}
```

`main_test.bal`
```ballerina
// Stubbing to return a specific value on a specific path parameter and method arguments
test:prepare(empClient)
    .whenResource("employee/welcome/:id")
    .onMethod("get")
    .withPathParameters({id: ""})
    .withArguments("", "").
    thenReturn("Stub_4");
```